deploy/examples/cluster-on-pvc-minikube.yaml

```diff
@@ -90,6 +95,31 @@ spec:
               values:
                 - minikube
 ---
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: local0-3
+spec:
+  storageClassName: local-storage
+  capacity:
+    storage: 20Gi
+  accessModes:
+    - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Retain
+  # PV for mon must be a filesystem volume.
+  volumeMode: Block
+  local:
+    # To use dm devices like logical volume, please replace `/dev/sdb` with their device names like `/dev/vg-name/lv-name`.
+    path: /dev/vde
```

Comment:
I don't know for sure, but these device IDs might change between different minikube drivers. hyperkit, qemu, kvm might not have the same disk IDs.

On qemu (mac M1), the last disk is the boot disk, vda is unusable, and usable disks begin with `vdb`. So `vdf` is the boot disk.

Reply:
@BlaineEXE good point, I didn't know that minikube devices naming was driver dependent. This file is meant to be used with kvm2 driver so I'm afraid that it could not work with other drivers. I'll add a note on the header to make that clear.

Reply:
I think that's good. Just note that it's developed/tested on kvm2 and that users might need to change the `vdN` assignments based on their minikube cluster. I'd say that in the header comment. And then a short comment in the static PV definitions next to `vd[bcde]` that they might need to change for minikube drivers that aren't kvm2 could be helpful.